storage/innobase/xtrabackup/src/xbcloud/s3.cc

```diff
@@ -742,6 +742,19 @@ void S3_client::retry_error(Http_response *resp, bool *retry) {
     if (s3_resp.error_code() == "RequestTimeout") {
       *retry = true;
     }
+    if (s3_resp.error_code() == "ExpiredToken" ||
+        s3_resp.error_code() == "InvalidToken" ||
+        s3_resp.error_code() == "TokenRefreshRequired") {
+      if (api_version == S3_V4 && ec2_instance != nullptr &&
```

Comment:
Will this support the parallel call of retry_error, `fetch_metadata` is not thread-safe. multiple threads could try to write into the access key. and xbcloud can crash. Can you please check

Reply:
xbcloud is not multi thread, it has an event handler that process up to --parallel async request at the same time. But it is all single thread.